draft-birkholz-cose-cometre-ccf-profile.md

```diff
@@ -41,38 +41,39 @@ normative:
+The Concise Encoding of Signed Merkle Tree Proofs (CoMeTre) {{-COMTRE}} defines a common framework for defining different types of proofs about verifiable data structures (also abbreviated as "logs" in this document). For instance, inclusion proofs guarantee to a verifier that a given serializable element is recorded at a given state of the log, while consistency proofs are used to establish that an inclusion proof is still consistent with the new state of the log at a later time.
 
-In this document, we define a new type of log, associated with the Confidential Consortium Framework (CCF) ledger. Compared to {{-certificate-transparency-v2}}, the leaves of CCF trees carry additional opaque information that is used to verify that elements are only written by the Trusted Execution Environment, which addresses the persistence of committed transactions that happen between new signatures of the Merkle Tree root.
+In this document, we define a new type of log, associated with the Confidential Consortium Framework (CCF) ledger. This log carries indexed transaction information in a binary Merkle Tree, where new transactions are appended to the right, so that the binary decomposition of the index of a transaction can be interpreted as the position in the tree if 0 represents the left branch and 1 the right branch.  Compared to {{-certificate-transparency-v2}}, the leaves of CCF trees carry additional opaque information that is used to verify that elements are only written by the Trusted Execution Environment, which addresses the persistence of committed transactions that happen between new signatures of the Merkle Tree root.
 
 ## Requirements Notation
 
 {::boilerplate bcp14-tagged}
 
-# Description of the CCF Ledger Verifiable Data Structure
+# Format of CCF Ledger
 
 This documents extends the verifiable data structure registry of {{-COMTRE}} with the following value:
 
-| Identifier            | Algorithm | Reference
+| Name            | Value   | Description | Reference
 |---
-|TBD_1 | CCF_LEDGER     | This document
+|CCF_LEDGER_SHA256 | TBD_1 (requested assignment 2) | Historical transaction ledgers, such as the CCF ledger | This document
 {: #verifiable-data-structure-values align="left" title="Verifiable Data Structure Algorithms"}
 
-## Tree Shape
+This document defines inclusion proofs and consistency proof formats for CCF ledgers. Verifiers MUST reject all other proof types.
 
-The input of the Merkle Tree Hash (MTH) function is a list of n byte strings, written D_n = \{d\[0\], d\[1\], ..., d\[n-1\]\}. The output is a single HASH_SIZE byte string, also called the Merkle root hash.
+## Merkle Tree Shape
 
-This function is defined as follows:
+A CCF ledger is a binary Merkle Tree constructed from a hash function H, which is defined from the log type. For instance, the hash function for `CCF_LEDGER_SHA256` is `SHA256`, whose `HASH_SIZE` is 32 bytes.
```

Comment:
Good for now. A pre-existing external references to back that description would bolster that assertion.